### DISCUSSION TITLE
Fedora and CentOS 7 support dropped based on EOL, despite neither being EOL

### DISCUSSION THREAD

**Discussion by tjmoore** [PowerShell/PowerShell#17574] [General]
Carrying over from an issue I raised on documentation having dropped all instructions for CentOS and Fedora: https://github.com/MicrosoftDocs/PowerShell-Docs/issues/8954

Response is that documentation has been updated to remove CentOS & Fedora as support has been dropped based on PowerShell team policy, based on End-of-Life dates.

It's confusing for users with existing PowerShell installs on these platforms that get prompts that a new version is available, but the link takes you to documentation stating:

"CentOS and Fedora distributions are no longer supported. The versions of these operating systems that were supported have reached their end-of-life dates. We are not supporting any newer versions."

and all existing instructions have been removed.

While support may have been dropped, the statement they they have reach end-of-life is wrong. Fedora is definitely not End-of-Life with ongoing development of 35 & 36 with 37 due later in the year. Fedora is the upstream of CentOS Stream and RHEL, and that while CentOS 8* is EOL, CentOS 7 is still in active support until 2024.

https://endoflife.date/centos
https://endoflife.date/fedora
https://en.wikipedia.org/wiki/Fedora_Linux_release_history

\* - From .NET docs: "CentOS 8 reached an early End Of Life (EOL) on December 31st, 2021". Noting .NET supports CentOS 7 and Fedora (but not CentOS 8 due to EOL).

**Comment by kasini3000** [comment]
More to help promote this third-party one-click installation powershell project, don't worry too much about that document.

----

[kasini3000_agent_linux](https://gitee.com/chuanjiao10/kasini3000_agent_linux) Shell script,one click install powershell on linux,modify sshd_config for PsRemote.
